guides/common/modules/proc_salt-configuring-salt.adoc

```diff
@@ -0,0 +1,16 @@
+[id="salt_guide_configuring_salt_{context}"]
+= Configuring Salt
+
+Once you've installed the Salt plugin, you need to connect it to a Salt Master.
```

Comment:
I find this stuff interesting, I am unsure if anyone else cares, but for translation, this can be helpful for machine translation. This is from IBM Style: 

once adv
Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”

In my previous job, I had a lovely book https://www.oreilly.com/library/view/the-global-english/9781599946573/ that explained things well with regards to technical translation. I am unsure if anyone cares and consider this a non-blocking comment also.

Reply:
I need to really read IBM Style now.

Reply:
Replaced with `after`.